src/docs/sdk/unified-api/tracing.mdx

```diff
@@ -43,6 +43,12 @@ Apart from that, the `Event` gets new fields: `spans`, `contexts.TraceContext`.
 
 ## New `Span` and `Transaction` Classes
 
+In memory, spans build-up a conceptual tree of timed operations. We call the whole span tree a transaction. So one can think of the transaction as either the whole tree or the root span of the tree. In inheritance terms, `Transaction` "is a (subclass of)" `Span`.
```

Comment:
> In inheritance terms, `Transaction` "is a (subclass of)" `Span`.

Is this an implementation detail, or part of the spec? Would a `Transaction` class that didn't inherit but instead had a `rootSpan` property (but otherwise had the same API as we currently have) be to spec or not?

Reply:
Some paragraphs later in this page:

> The `Transaction` class may inherit from `Span`, but that's an implementation detail.

The intention of the paragraph I'm adding is to give a high-level overview of the concepts, yet without going too deep -- this paragraph is just copy-paste from me explaining the difference in a PR comment that would get lost otherwise :)

Reply:
```suggestion
In memory, spans build-up a conceptual tree of timed operations. We call the whole span tree a transaction. In many SDKs, the transaction is represented by its root span (with a few extra attributes added in), but that is not a required implementation.
```

Reply:
If talking about inheritance is confusing, I'd remove the last sentence as it is not the critical piece here.

```suggestion
In memory, spans build up a conceptual tree of timed operations. We call the whole span tree a transaction. Sometimes we use the term "transaction" to refer to a span tree as a whole tree, sometimes to refer specifically to the root span of the tree.
```

@lobsterkatie would that work for you?

Reply:
Sure, sounds great.